pkgs/tools/typesetting/tex/auctex/default.nix

```diff
@@ -1,41 +0,0 @@
-{ lib, stdenv, fetchurl, emacs, texliveBasic, ghostscript }:
-
-let auctex = stdenv.mkDerivation ( rec {
-  # Make this a valid tex(live-new) package;
-  # the pkgs attribute is provided with a hack below.
```

Comment:
Without this PR, users can build a tex environment containing auctex.  With this PR applied, presumably that can not be done easily.  Maybe we should update emacsPackages.auctex to make that easy.  Here is the doc: https://nixos.org/manual/nixpkgs/stable/#sec-language-texlive-custom-packages